deploy/csi-controller-deployment.yaml

```diff
@@ -14,16 +14,16 @@ spec:
         app: cfs-csi-controller
     spec:
       serviceAccount: cfs-csi-service-account
+      serviceAccountName: cfs-csi-service-account
       nodeSelector:
         cubefs-csi-controller: enabled
       containers:
         - name: csi-provisioner
-          image: quay.io/k8scsi/csi-provisioner:v1.6.0
+          image: k8s.gcr.io/sig-storage/csi-provisioner:v2.2.2
```

Comment:
all this official csi image have already completely tested?
which k8s minimum version of this patch support? 1.15+?

Reply:
yes, 1.17+ is required